### ISSUE TITLE
Remove update value in getter

### ISSUE THREAD

**Issue by kocunop** [mikeengland/django-setty#14]
Hello!

backend.py
```python
def _retrieve_and_cache_setting(self, name):
        value = super().get(name)
        return self.set(name, value)
``` 
No need to update the value in getter. 

` return self.set(name, value) -> return self.set_in_cache(name, value)`???

**Comment by mikeengland** [comment]
Makes sense, will take a look!

**Comment by mikeengland** [comment]
Published as version 3.2.2
